packages/components/organisms/f-cookie-banner/test/accessibility/axe-accessibility.spec.js

```diff
@@ -28,12 +22,12 @@ describe('Legacy Accessibility tests', () => {
 
     it('a11y - should test the f-cookie-banner component WCAG compliance', () => {
         // Arrange
-        cookieConsentBanner = new CookieConsentBanner();
+        const cookieConsentBanner = new CookieConsentBanner();
         cookieConsentBanner.withQuery('args', 'locale:en-GB');
```

Comment:
minor - `withQuery` function here and in the test below to be deleted (should fix the failing tests) 👍